### PR TITLE
Alt, anonymized anti-FUD post

### PR DESCRIPTION
**Pull request by sicarie**
Alt version of anti-FUD post to address FUD in general

---

This change is [Reviewable](https://reviewable.io/reviews/openstack-security/openstack-security.github.io/23)

### CODE REVIEW

**Comment by mcpeak** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 18 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5pbNb4H3ecnZzt7os:-KH5pbNb4H3ecnZzt7ot:890918151) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L18)):_
This sentence is a little long.  Maybe:

Some of these presentations call out important concepts such as... .  Unfortunately they also highlight vulnerabilities or attack vectors... responsibility of the the group deploying and maintaining the cloud.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 41 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5qLJUKwzSv2zMCoXh:-KH5qLJUKwzSv2zMCoXi:199416987) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L41)):_
*components

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 45 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5qSRaDt-jxLcqi2Fp:-KH5qSRaDt-jxLcqi2Fq:-1544756773) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L45)):_
remove "so"

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 48 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5qVMtf5JR3RXTDxsB:-KH5qVMtf5JR3RXTDxsC:-1079960462) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L48)):_
I think it's PostgreSQL ?

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23)_

**Comment by hyakuhei** [comment]
Review status: all files reviewed at latest revision, 7 unresolved discussions.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 21 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5tSAAZCZrX4RnrJ3P:-KH5tSAAZCZrX4RnrJ3Q:-1829491403) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L21)):_
The problem I have here is that upstream OpenStack in not demonstrably secure. It's better than has been implied recently though. Perhaps "To understand how to handle OpenStack vulnerabilties securely" or some shit. I mean the existence of a VMT means something isn't completely secure

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 41 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5u0M0W76i2Xkqz8B6:-KH5u0M1z0OXmJD6Xk6I:-1871325733) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L41)):_
components

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 42 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5u4iBVXVayVh3T9BB:-KH5u4iBVXVayVh3T9BC:1866846404) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L42)):_
portion of recently discussed "vulnerabilities" 

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 48 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5qVMtf5JR3RXTDxsB:-KH5uC8xTkFbOWD8bgkN:5172523) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L48)):_
+1

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 55 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5uFXIKBsHWa-4z0is:-KH5uFXIKBsHWa-4z0it:915681311) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L55)):_
Great to see this here but you shouldn't introduce new things in a conclusion

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 57 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5uIumGDNc9Op7EHf2:-KH5uIumGDNc9Op7EHf3:193480961) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L57)):_
Again, don't introduce new things in a conclusion. It should be used to tie together points you've discussed in more detail.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23)_

**Comment by sicarie** [comment]
Review status: 0 of 1 files reviewed at latest revision, 8 unresolved discussions.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 18 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5pbNb4H3ecnZzt7os:-KH5v8emP9AXKwbdbYWM:-499146009) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L18)):_
Done.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 21 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5tSAAZCZrX4RnrJ3P:-KH5v9JdOkYbwANGAGi-:-499146009) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L21)):_
Done.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 41 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5u0M0W76i2Xkqz8B6:-KH5vIOUMKFv3DCx-Ce4:-499146009) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L41)):_
Done.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 42 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5u4iBVXVayVh3T9BB:-KH5vO8n97pcFgbkxXF-:-499146009) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L42)):_
Done.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 45 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5qSRaDt-jxLcqi2Fp:-KH5vRukf4mK_HpLhKc7:-499146009) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L45)):_
Done.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 48 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5qVMtf5JR3RXTDxsB:-KH5vSHFpfXQDRPpOHMM:-499146009) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L48)):_
Done.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23)_

File: _posts/2016-05-05-clearing-the-air.markdown [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+layout: post
+title:  "Clearing the Air about Vulnerabilities in OpenStack"
+date:   2016-05-05
+categories: vulnerabilities
+tags: security vmt vulnerabilities summit
+author: "tmcpeak @ IBM Cloud, sicarie @ HPE"
+---
+
+Recently, there have been a few talks around "vulnerabilities" within
+the OpenStack project that missed the mark so much they deserve note.
```
Comment (Daviey) [inline _posts/2016-05-05-clearing-the-air.markdown]: "missed the mark so much" probably isn't the right tone.  Maybe, "introduced some unfair concern"

File: _posts/2016-05-05-clearing-the-air.markdown [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+deploying and maintaining the cloud.
+
+To understand how OpenStack handles vulnerabilities securely, it is
+important to briefly introduce the OpenStack vulnerability management
+process.
+
+## Vulnerability Management in OpenStack
+A vulnerability in OpenStack usually begins life as a bug filed against
+a project with the "security" tag.  These bugs are marked private and sent
+directly to the project's security team and the VMT.  An initial triage is
```
Comment (Daviey) [inline _posts/2016-05-05-clearing-the-air.markdown]: "VMT", first use probably requires writing out in full.. and maybe a link to https://security.openstack.org/vmt-process.html

File: _posts/2016-05-05-clearing-the-air.markdown [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+performed to understand whether the bug represents a legitimate security issue
+and if so what the impact is.  If the issue is confirmed, an advisory and patch
+are prepared and validated privately.  Once the the advisory and fix are
+available, OpenStack stakeholders are given two weeks early notice to patch
```
Comment (Daviey) [inline _posts/2016-05-05-clearing-the-air.markdown]: Should probably write out, that "distributions and vendors" (where the majority of deployers base their installations on)

File: _posts/2016-05-05-clearing-the-air.markdown [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## Vulnerability Management Outside of OpenStack
+The Vulnerability Management Team (VMT) only manages issues for OpenStack
+components (with the "vulnerability:managed" tag) inside the OpenStack ecosystem.
+A portion of recently discussed "vulnerabilities" have to do with third party
+applications deployed on an OpenStack cloud. Unfortunately, much as an
+operating system is utilized to host insecure things like old versions of
+Java or PHP that can be vulnerable to many attacks, OpenStack is used to
+host applications that can also be vulnerable. It is the responsibility of
+the group maintaining and distributing the third party application to managed
+the security of that application (ie, a vulnerability in PostgreSQL is `not`
```
Comment (Daviey) [inline _posts/2016-05-05-clearing-the-air.markdown]: This does sound a bit like we are saying, "yeah, it has issues but aint our problem gov'".. but in honestly, the responsibility is shared amongst the upstream software project, OSSG (with suitable hardening guidelines) , distributions (to make sure they have sane defaults) and deployers to make sure they have safe configuration and regular patching.

File: _posts/2016-05-05-clearing-the-air.markdown [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+[OpenStack Security Guide](http://docs.openstack.org/security-guide/) are
+frequently summarized with checklists [such as this one on the Identity service,
+Keystone](http://docs.openstack.org/security-guide/identity/checklist.html) to
+help deployers stand up a service securely.  Additionally, the OpenStack Security
+Project also makes [Security Notes](https://wiki.openstack.org/wiki/Security_Notes)
+available to provide quick security guidance to deployers.
+
+Finally, if you have found a security issue in OpenStack, please disclose it
+responsibly by marking "security" in the LaunchPad bug report, or by contacting
+the involved project team or VMT team members directly.
```
Comment (Daviey) [inline _posts/2016-05-05-clearing-the-air.markdown]: We should probably mention that OSSG welcomes involvement and happy to discuss concerns etc.

**Comment by mcpeak** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 8 unresolved discussions.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23)_

**Comment by mcpeak** [comment]
Review status: all files reviewed at latest revision, 7 unresolved discussions.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 55 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5uFXIKBsHWa-4z0is:-KHzEU-T_xyy865Tl3_C:5173665) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L55)):_
<details><summary>

<i>Previously, hyakuhei (Robert) wrote…</i></summary>



> Great to see this here but you shouldn't introduce new things in a conclusion</details>
> OK

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 57 [r1]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH5uIumGDNc9Op7EHf2:-KHzEYlnS-WQftjgtocp:5173665) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/a0cf2680377aa7e5d23937d1cf926c41c2152afe/_posts/2016-05-05-clearing-the-air.markdown#L57)):_
<details><summary>

<i>Previously, hyakuhei (Robert) wrote…</i></summary>



> Again, don't introduce new things in a conclusion. It should be used to tie together points you've discussed in more detail.</details>
> OK

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 65 [r2]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH2a4vAxCHnrhhFcJjw-r2-65:-KHzE_L4woFTOrRDgqzc:5173665) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/fa1344cbf3a9bbeb6b8964da2533ef7eb57a91d0/_posts/2016-05-05-clearing-the-air.markdown#L65)):_
<details><summary>

<i>Previously, Daviey (Dave Walker) wrote…</i></summary>



> We should probably mention that OSSG welcomes involvement and happy to discuss concerns etc.</details>
> OK

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23)_

**Comment by mcpeak** [comment]
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 11 [r2]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH2a4vAxCHnrhhFcJjw-r2-11:-KHzF5QFaKjku7wdvMSU:5173665) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/fa1344cbf3a9bbeb6b8964da2533ef7eb57a91d0/_posts/2016-05-05-clearing-the-air.markdown#L11)):_
<details><summary>

<i>Previously, Daviey (Dave Walker) wrote…</i></summary>



> "missed the mark so much" probably isn't the right tone.  Maybe, "introduced some unfair concern" </details>
> OK

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 28 [r2]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH2a4vAxCHnrhhFcJjw-r2-28:-KHzF6M1rlj1keYS9geN:5173665) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/fa1344cbf3a9bbeb6b8964da2533ef7eb57a91d0/_posts/2016-05-05-clearing-the-air.markdown#L28)):_
<details><summary>

<i>Previously, Daviey (Dave Walker) wrote…</i></summary>



> "VMT", first use probably requires writing out in full.. and maybe a link to https://security.openstack.org/vmt-process.html</details>
> OK

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 32 [r2]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH2a4vAxCHnrhhFcJjw-r2-32:-KHzF70KvjXnufARzqox:5173665) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/fa1344cbf3a9bbeb6b8964da2533ef7eb57a91d0/_posts/2016-05-05-clearing-the-air.markdown#L32)):_
<details><summary>

<i>Previously, Daviey (Dave Walker) wrote…</i></summary>



> Should probably write out, that "distributions and vendors" (where the majority of deployers base their installations on)</details>
> OK

---

_[_posts/2016-05-05-clearing-the-air.markdown, line 49 [r2]](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23#-KH2a4vAxCHnrhhFcJjw-r2-49:-KHzF8_6eC-NW5C17StP:5173665) ([raw file](https://github.com/openstack-security/openstack-security.github.io/blob/fa1344cbf3a9bbeb6b8964da2533ef7eb57a91d0/_posts/2016-05-05-clearing-the-air.markdown#L49)):_
<details><summary>

<i>Previously, Daviey (Dave Walker) wrote…</i></summary>



> This does sound a bit like we are saying, "yeah, it has issues but aint our problem gov'".. but in honestly, the responsibility is shared amongst the upstream software project, OSSG (with suitable hardening guidelines) , distributions (to make sure they have sane defaults) and deployers to make sure they have safe configuration and regular patching.</details>
> OK

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openstack-security/openstack-security.github.io/23)_

**Pull request merged**